indra/llmath/llmath.h

```diff
@@ -169,12 +169,12 @@ inline S32 lltrunc( F32 f )
 #endif
 }
 
-inline S32 lltrunc( F64 f )
+constexpr S32 lltrunc(F64 f)
 {
     return (S32)f;
```

Comment:
Maybe `narrow(f)` ?

Reply:
This kinda sounds like replacing `lltrunc` entirely.